File: java_backup/my java/romit/pattern2.java
import java.io.*;
import java.lang.*;
class pattern2
{
public static void main(String args[])throws IOException
{
BufferedReader Input=new BufferedReader(new InputStreamReader(System.in));
System.out.println("ENTER THE NUMBER");
int n=Integer.parseInt(Input.readLine());
int i,j,m=1;
for(i=1;i<=n;i++)
{
for(j=1;j<=i;j++)
{
System.out.print(m);
}
System.out.println(' ');
m++;
}
}
}

File: c++_backup/Amlyan/c++/pattern2.cpp
#include<iostream>
#include<cmath>
using namespace std;
class pattern2
{
	int n;
	public:
		void input()
		{
			cout<<"Enter a number";
			cin>>n;
		}
		void pattern()
		{
			for(int i=1;i<=n;i++)
			{
				cout<<i<<" ";
			}
			cout<<"\n";
			for(int i=2;i<n;i++)
			{
				cout<<i<<"  ";
				for(int j=2;j<n;j++)
				{
					cout<<"  ";
				}
				cout<<"\b"<<(n-i+1);
				cout<<"\n";
			}
			for(int i=n;i>0;i--)
			{
				cout<<i<<" ";
			}
		}
};
int main()
{
	pattern2 p;
	p.input();
	p.pattern();
}
